src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -418,7 +418,7 @@ def GetDpaProtectedChannels(freq_ranges_mhz, is_portal_dpa=False):
   channels = set()
   for freq_range in freq_ranges_mhz:
     min_freq = int(freq_range[0] / DPA_CHANNEL_BANDWIDTH) * DPA_CHANNEL_BANDWIDTH
-    max_freq = int(freq_range[1] / DPA_CHANNEL_BANDWIDTH) * DPA_CHANNEL_BANDWIDTH
+    max_freq = freq_range[1]
```

Comment:
why we do not need to do the same for min_freq, does this assume that min_freq  is always on the DPA_CHANNEL_BANDWIDTH boundary ?